ale/drivers/mro_drivers.py

```diff
@@ -39,27 +42,25 @@ def spacecraft_id(self):
         : int
           Naif ID code for the spacecraft
         """
-        return "N/A"
+        return "-74"
```

Comment:
Is this not in the kernels?

Reply:
The `spice.bods2c(self.spacecraft_name)` call in data_naif should cover this if it's in the kernels as @Kelvinrr pointed out. Is the spacecraft_name not returning the correct value?

Reply:
Does this work on the ISIS side? I thought we didn't load any kernels?

Reply:
No this doesn't work without kernels sorry!

Reply:
I think we could get this from one of the frame codes but it would be a little hacky